src/main/java/com/conveyal/gtfs/validator/ReversedTripsValidator.java

```diff
@@ -36,21 +39,20 @@ public boolean validate(GTFSFeed feed, boolean repair, Double distanceMultiplier
     @Override
     public boolean validate(GTFSFeed feed, boolean repair) {
         boolean isValid = true;
-        ValidationResult result = new ValidationResult();
         int errorLimit = 5000;
```

Comment:
Ah, just to avoid spewing too many errors. I think we shoudn't do this, errors should be filtered at display time but we shouldn't have hard limits on how many are generated.